docs/source/getting-started-faqs.md

```diff
@@ -0,0 +1,54 @@
+# FAQs and Common Issues
+
+This page summarizes frequently asked questions and provides guidance on issues that commonly occur when adopting ExecuTorch.
+
```

Comment:
Can you also remind the user to check https://github.com/pytorch/executorch/issues and https://github.com/pytorch/executorch/discussions

Reply:
Good idea. I will add that as a follow-up today.